src/cdk/testing/tests/env-independent/shared-specs.ts

```diff
@@ -0,0 +1,514 @@
+/**
```

Comment:
Could you split this PR into 2 commits, like this:
- commit 1 - exactly what you have in this PR now, just replace the contents of this file with the contents of `src/cdk/testing/tests/testbed.spec.ts` (from master)
- commit 2 - change the contents of this file back to what they currently are

I'm hoping that will result in a better diff so I can see what actually changed

Reply:
done.